dkg-gadget/src/lib.rs

```diff
@@ -122,6 +122,9 @@ where
 	C: Client<B, BE> + 'static,
 	C::Api: DKGApi<B, AuthorityId, NumberFor<B>>,
 {
+	// ensure logging-related statics are initialized
+	dkg_logging::setup_log();
```

Comment:
I do believe this will fail here, maybe I'm not sure, but we can't setup a global subscriber, since substrate already register a global one, in the node's CLI when it runs.

Reply:
Per our discussion, we could just use the built-in env-filter+subscriber that is used with substrate to send logging events to the console.